### PR TITLE
Defect fix for the virtual machine power off

### PR DESCRIPTION
**Pull request by KalpanaBhardwaj**
Signed-off-by: KalpanaBhardwaj <bhardwaj@hpe.com>

### Description
To return true or false for VM "power_off" method to be consistent with "power_on" method.

### Issues Resolved
fixing defect #49 

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
     - required __init__.py files to support unit test discovery
     - resolve failing unit test cases
     - resolve the virtual machine `__refresh` method
+    - resolve the virtual machine `power_off` method to be consistent with power_on
```
Comment (svt-hchaudhari) [inline CHANGELOG.md]: Need to remove this line from CHANGELOG.md, as power_off api is not released in the earlier version

File: examples/virtual_machines.py [hunk 2]
```diff
@@ -171,7 +171,8 @@
 print("\nPower off")
```
Comment (svt-hchaudhari) [inline examples/virtual_machines.py]: Usually for setup we using ("\n\npower off"), Please execute the example file and check if all prints are seen correctly

File: CHANGELOG.md (hunk 1 above)
Comment (KalpanaBhardwaj) [reply CHANGELOG.md]: Thanks Hemant,

Updated.

File: examples/virtual_machines.py (hunk 2 above)
Comment (KalpanaBhardwaj) [reply examples/virtual_machines.py]: Done.

**Review by svt-hchaudhari — APPROVED** [review]

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**